models/standalone-price/src/transformers.ts

```diff
@@ -14,19 +14,11 @@ import type {
 
 // Overloads
 function createCustomerGroupReference(
```

Comment:
Good catch.
I wasn't sure tbh but it seems it was a type safety thing because the function was handling 2 different targets and types of output.
Not needed anymore